File: Python/Program/Mahasiswa.py
class Mahasiswa:
    __nama = ""
    __nim = ""
    __prodi = ""
    __fakultas = ""

    def __init__(self, nama = "", nim = "", prodi = "", fakultas = ""):
        self.__nama = nama
        self.__nim = nim
        self.__prodi = prodi
        self.__fakultas = fakultas

    # Get-set Nama
    def setNama(self, nama):
        self.__nama = nama

    def getNama(self):
        return self.__nama
    
    # Get-set NIM
    def setNIM(self, nim):
        self.__nim = nim

    def getNIM(self):
        return self.__nim

    # Get-set Prodi
    def setProdi(self, prodi):
        self.__prodi = prodi

    def getProdi(self):
        return self.__prodi

    # Get-set Fakultas
    def setFakultas(self, fakultas):
        self.__fakultas = fakultas

    def getFakultas(self):
        return self.__fakultas

File: PHP/Program/Mahasiswa.php
<?php

class Mahasiswa
{
    private $nama = "";
    private $nim = "";
    private $prodi = "";
    private $fakultas = "";

    public function __construct($nama = "", $nim = "", $prodi = "", $fakultas = ""){
        $this->nama = $nama;
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
    }

    // Get-set Nama
    public function setNama($nama){
        $this->nama = $nama;
    }

    public function getNama(){
        return $this->nama;
    }

    // Get-set NIM
    public function setNIM($nim){
        $this->nim = $nim;
    }

    public function getNIM(){
        return $this->nim;
    }

    // Get-set Prodi
    public function setProdi($prodi){
        $this->prodi = $prodi;
    }

    public function getProdi(){
        return $this->prodi;
    }

    // Get-set Fakultas
    public function setFakultas($fakultas){
        $this->fakultas = $fakultas;
    }

    public function getFakultas(){
        return $this->fakultas;
    }
}

?>